lumicks/pylake/nb_widgets/correlated_plot.py

```diff
@@ -46,10 +47,16 @@ def plot_correlated(
         Whether plots should be aligned vertically.
     return_handle : bool
         Whether to return a handle to the update function.
+    downsample_channel : bool
+        Downsample the channel data over the frame (default: True).
     """
     import matplotlib.pyplot as plt
 
-    downsampled = channel_slice.downsampled_over(frame_timestamps, where="left", reduce=reduce)
+    downsampled = (
+        channel_slice.downsampled_over(frame_timestamps, where="left", reduce=reduce)
+        if downsample_channel
+        else channel_slice[frame_timestamps[0][0] : frame_timestamps[-1][-1]]
+    )
```

Comment:
nitpick: I wonder if it'd be clearer to overwrite `channel_slice` here since `downsampled` doesn't necessarily make sense anymore?

Reply:
I also doubted about this. I'm not a huge fan of the word downsample anymore, but we can't replace it since for the downsampled case, we need the original channel once below for the last frame.

How about `processed_channel`? I could also go for `downsampled_or_sliced_channel` but it feels painfully verbose.